packages/datepicker-react/src/types.ts

```diff
@@ -30,6 +30,8 @@ export type DatePickerMetadata = {
     value: string;
 };
 
+export type YearsToShow = number | "all" | { previous: number; coming: number };
```

Comment:
Ikke veldig viktig, men _elsker_ ikke `previous` og `coming` som navn 😅 Tenkte først `past` og `future`, men utvalget av år er jo en _moving target_, så alt kan i teorien være i fortiden. Går an å ta inspirasjon fra browser-API som [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) og si `start` og `end`, men er ikke kjempefornøyd med det heller.

Navn, altså 🙈 Igjen, ikke så viktig, så vi kan godt gå for `previous` og `coming`.

Reply:
Hehe, jeg slet også med disse navnene 😅